labs/behaviors/focusable.ts

```diff
@@ -77,10 +77,12 @@ export function mixinFocusable<T extends MixinBase<LitElement>>(
     constructor(...args: any[]) {
       super(...args);
       this[isFocusable] = true;
-      queueMicrotask(() => {
-        this[hasConstructed] = true;
-        this[updateTabIndex]();
-      });
+    }
+
+    connectedCallback() {
+      super.connectedCallback();
+      this[hasConstructed] = true;
```

Comment:
I think we can completely remove the `hasConstructed` property, symbol, and the constructor by changing `privateIsFocusable` to default to false.

```ts
    [privateIsFocusable] = true;
    [externalTabIndex]: number | null = null;
    [isUpdatingTabIndex] = false;

    // * also don't forget `override`!
    override connectedCallback() {
      super.connectedCallback();
      this[updateTabIndex]();
    }
```

We needed `hasConstructed` because we set `isFocusable` in the constructor, which called `updateTabIndex`.

If we bypass `isFocusable = true` and directly set `privateIsFocusable = true`, then we don't need to do that dance.

Reply:
Seems fine to me, as such having Microtask or setTimeout can end up too early or too short giving timing problems anyway in all sorts of scenarios.